examples/devices/sensor_tag_ble_dell5k_mqtt_iotcc.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------#
```

Comment:
We have separate folder for model_simulated examples.  This is the first actual device.  DHT examples can go in here.